### ISSUE TITLE
CRF intsead of softmax?

### ISSUE THREAD

**Issue by samarohith** [murthyrudra/NeuralNER#8]
I want to use CRF instead of softmax(logsoftmax). How would you recommend doing it?  There is a library "Pytorch-crf" available.

**Comment by murthyrudra** [comment]
Hi, Sorry for the very late reply. I tried adding CRF layer but the module kept on failing.
